netifaces.c

```diff
@@ -1,18 +1,5 @@
 #include <Python.h>
 
-/* Before Python 2.6, PyUnicode_FromString doesn't exist */
-#if PY_MAJOR_VERSION < 2 || (PY_MAJOR_VERSION == 2 && PY_MINOR_VERSION < 6)
-PyObject *PyUnicode_FromString(const char *s)
-{
-  Py_ssize_t len = strlen(s);
-  if (!len) {
-    Py_UNICODE uc = 0;
-    return PyUnicode_FromUnicode(&uc, 0);
-  }
-  return PyUnicode_DecodeUTF8(s, len, NULL);
-}
-#endif
-
```

Comment:
I don't see a good reason for removing this code. I agree that few people are probably using Python 2.5 or earlier any more, but this doesn't break anything for later versions.